source/guides/maintenance/developer-workstation.rst

```diff
@@ -17,67 +17,204 @@ other Linux\* distributions.
+           - `machine-learning-basic <https://clearlinux.org/software/bundle/machine-learning-basic/>`_
+
+         * - Build machine learning applications with PyTorch, an optimized tensor library for deep learning.
+           - `machine-learning-pytorch <https://clearlinux.org/software/bundle/machine-learning-pytorch/>`_
+
+         * - Build machine learning applications using Tensorflow, a library for numerical computation using deep neural networks.
+           - `machine-learning-tensorflow <https://clearlinux.org/software/bundle/machine-learning-tensorflow/>`_
+
+         * - Web-based, interactive tools for machine learning.
+           - `machine-learning-web-ui <https://clearlinux.org/software/bundle/machine-learning-web-ui/>`_
+
+         * - Machine learning Docker container.
+           - `machine-learning <https://clearlinux.org/software/docker/machine-learning/>`_
+
+         * - Pre-built Python libraries for Data Science.
+           - `python-extras <https://clearlinux.org/software/bundle/python-extras>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+   .. tab:: Computer Vision Engineer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Build computer vision applications.
+           - `computer-vision-basic <https://clearlinux.org/software/bundle/computer-vision-basic/>`_
+
+         * - Work with deep learning and edge-optimized models.
+           - `computer-vision-models <https://clearlinux.org/software/bundle/computer-vision-models/>`_
+
+         * - Basic OpenVINO™ toolkit.
+           - `computer-vision-openvino <https://clearlinux.org/software/bundle/computer-vision-openvino/>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+         * - All content for pkgconfig file opencv.pc.
+           - `devpkg-opencv <https://clearlinux.org/software/bundle/devpkg-opencv/>`_
+
+   .. tab:: Cloud Orchestration Engineer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Contains Clear Linux\* OS native software for cloud.
+           - `ethtool <https://clearlinux.org/software/bundle/ethtool/>`_
+
+         * - Utilities for controlling TCP/IP networking and traffic control.
+           - `iproute2 <https://clearlinux.org/software/bundle/iproute2/>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+         * - C++ runtime support.
+           - `libstdcpp <https://clearlinux.org/software/bundle/libstdcpp/>`_
+
+         * - Load and enumerate PKCS#11 modules.
+           - `p11-kit <https://clearlinux.org/software/bundle/p11-kit/>`_
+
+   .. tab:: Game Developer
```

Comment:
@ahkok Ditto for this profile as for which bundles are appropriate.

Reply:
@ahkok. Do you have time to give feedback on this?

Reply:
I would remove the "Game developer" section. We do not offer enough substance to make this warranted.

Reply:
I removed `Game Developer` in my last commit. Thanks for the feedback.